src/textures/VideoTexture.js

```diff
@@ -42,9 +31,31 @@ class VideoTexture extends Texture {
 		const video = this.image;
 		const hasVideoFrameCallback = 'requestVideoFrameCallback' in video;
 
-		if ( hasVideoFrameCallback === false && video.readyState >= video.HAVE_CURRENT_DATA ) {
+		if ( hasVideoFrameCallback === false ) {
+
+			if ( video.readyState >= video.HAVE_CURRENT_DATA ) {
+
+				this.needsUpdate = true;
```

Comment:
Because `update` is now called at construction, if there is no RVFC, this line can set `needsUpdate` to `true` just when the texture is created (which shouldn't break anything, I think, but unsure). Can fix this if desired.